_posts/en/newsletters/2021-06-16-newsletter.md

```diff
@@ -0,0 +1,134 @@
+    [short history][wuille taproot] of taproot's origins and history by
+    developer Pieter Wuille.
+
+- **BIP proposed for wallets to set nSequence by default on taproot transactions:**
+  Chris Belcher [posted][belcher post] a draft BIP to the Bitcoin-Dev
+  mailing list suggesting an alternative way wallets can implement [anti
+  fee sniping][topic fee sniping].  The alternative method would enhance
+  the privacy and fungibility of transactions made by single-sig users,
+  [multisignature][topic multisignature] users, and users of certain
+  contract protocols such as taproot-enabled LN or advanced
+  [coinswaps][topic coinswap].
+
+    Anti fee sniping is a technique some wallets implement to discourage
+    miners from trying to steal fees from each other in a way that would
+    reduce the amount of proof of work expended on securing Bitcoin and
+    limit users' ability to rely on confirmation scores.  All wallets
+    that implement anti fee sniping today use nLockTime height locks,
+    but it's also possible to implement the same protection using
+    [BIP68][] nSequence height locks.  This wouldn't be any more
+    effective at preventing fee sniping, but it would provide a good
+    reason for regular wallets to set their nSequence values to the same
+    values that are required for transactions in certain
+    multisignature-based contract protocols, such as ideas for coinswaps
+    and taproot-enabled LN.  This helps make regular wallet transactions
+    look like contract protocol transactions and vice versa.
+
+    Belcher's proposal suggests wallets randomly choose between using
+    either nLockTime or nSequence with 50% probability when both options
+    are available.  Overall, if the proposal is implemented, it will
+    allow users of regular single-sig transactions or uncomplicated
+    multisignatures to join together with users of contract protocols to
+    mutually improve each others' privacy and fungibility.
+
+## FIXME:article
+
+FIXME:harding to rebase on #582 on Tuesday
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Rust Bitcoin 0.26.2][] is the project's latest minor release.
+  Compared to the previous major version, it contains a several API
+  improvements and bug fixes.  See the [changelog][rb changelog] for
+  details.
+
+- [Rust-Lightning 0.0.98][] is a minor release containing several
+  improvements and bug fixes.  <!-- there's no release notes or
+  changelog I can see, so not much to say here. -->
+
+- [LND 0.13.0-beta.rc5][LND 0.13.0-beta] is a release candidate that
+  adds support for using a pruned Bitcoin full node, allows receiving
+  and sending payments using Atomic MultiPath ([AMP][topic multipath payments]),
+  and increases its [PSBT][topic psbt] capabilities, among other improvements
+  and bug fixes.
```

Comment:
Also included in the week before that.  I don't usually change the text between RCs unless something notable happens.  In this case, they didn't even change the RC number (which is fine IMO, especially since I think it's partly due to several people being in Miami).